plugins/package-managers/bazel/src/main/kotlin/BazelModel.kt

```diff
@@ -31,7 +31,9 @@ private val json = Json { ignoreUnknownKeys = true }
  */
 @Serializable
 internal data class Lockfile(
-    val flags: Flags? = null
+    val flags: Flags? = null,
+
+    val registryFileHashes: Map<String, String>? = null
```

Comment:
Please document these properties as well with the respective Bazel versions.

Reply:
Also, if only ever either or may be non-`null`, should an `init`-block with

    require(flags != null || registryFileHashes != null) { ... }

be introduced?

Reply:
Yes, but the test is more complicated as yours allow both of them to be non-null.